data/json/recipes/armor/boots.json

```diff
@@ -16,6 +16,23 @@
     ]
   },
   {
+    "result": "boots_chitin",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 6,
```

Comment:
It would be good to start providing top tier `tailor` recipes. There isn't much useful above 5?